loopd/run.go

```diff
@@ -149,13 +149,16 @@ func Run(rpcCfg RPCConfig) error {
 
 	// Parse ini file.
 	loopDir := lncfg.CleanAndExpandPath(config.LoopDir)
-	configFile := getConfigPath(config, loopDir)
+	configFile, explicitConfig := getConfigPath(config, loopDir)
```

Comment:
nit: `customConfig` makes a little more sense than `explicitConfig` to me, but nonblocking

Reply:
I'm not really sure. Each config is custom. :) Maybe `nonDefaultConfigPath`?

Reply:
maybe an action in the name? something like `hasExplicitConfig`,`hasConfigSet`, etc...